README.md

```diff
@@ -930,6 +931,30 @@ targets:
     license: MIT
 ```
 
+### Unity Package Manager (`upm`)
+
+Pulls the package as a zipped artefact and pushes the unzipped content to the target repository, tagging it with the provided version.
+
+_WARNING!_ The destination repository will be completely overwritten.
+
+**Environment**
+
+_none_
+
+**Configuration**
+
+| Option          | Description                           |
+| --------------- | ------------------------------------- |
+| `releaseTarget` | Name of the release target repository |
+
+**Example**
+
+```yaml
+targets:
+  - name: upm
+    releaseTarget: "unity"    
```

Comment:
I believe it's best to add a full URL here (not assume getsentry/$releaseTarget

Reply:
I've replaced "releaseTarget" with "releaseOwner" and "releaseRepo" instead.